src/app/components/create-user/create-user.component.ts

```diff
@@ -8,14 +9,42 @@ import {MatDialogRef} from "@angular/material";
 })
 export class CreateUserComponent implements OnInit {
 
+  username: string;
+  email: string;
+  password: string;
+  team: string;
   name: string;
 
-  constructor(public dialogRef: MatDialogRef<CreateUserComponent>,) { }
+  constructor(public dialogRef: MatDialogRef<CreateUserComponent>, private account: AccountService, private info: MatSnackBar) {
+  }
 
   ngOnInit() {
   }
 
-  save(){
+  signUp() {
+    let account = new Account(this.username, this.email, this.password, this.team);
+    this.account.signUp(account).subscribe(
+      data => {
+        this.account.account = account;
+        this.dialogRef.close(this.email);
+      },
+      error => this.info.open('Error, please try again later', '', {duration:3000})
```

Comment:
Consider extracting global const for cohesion (spójność, i had to look it up too :D ) of timeouts across the app

Reply:
Yea, great idea, I will fix it